### PR TITLE
[IMP] make update stock product selection modular and exclude by default...

### PR DESCRIPTION
**Pull request by bealdav**
... service product (ie magento bundle)

### CODE REVIEW

File: magentoerpconnect/magento_model.py [hunk 1]
```diff
@@ -273,14 +273,19 @@ def import_product_product(self, cr, uid, ids, context=None):
                                'import_products_from_date', context=context)
         return True
 
+    def update_stock_product_domain(self, cr, uid, ids, context=None):
```
Comment (guewen) [inline magentoerpconnect/magento_model.py]: The name of the method suggests it does an update. Also, it should be private I think.
My suggestion: `_domain_for_update_product_stock_qty`

**Comment by guewen** [comment]
Can you rebase please?

**Comment by bealdav** [comment]
@guewen rebase is done

**Comment by guewen** [comment]
Build failed:

> ./magentoerpconnect/magento_model.py:276:8: E271 multiple spaces after keyword
> ./magentoerpconnect/magento_model.py:276:80: E501 line too long (80 > 79 characters)

**Comment by guewen** [comment]
:+1:

**Comment by sebastienbeau** [comment]
LGTM :+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**